app/assets/javascripts/Measure.js

```diff
@@ -14,33 +14,29 @@ const [mNumber, mString, mBoolean, Mixed, ObjectId, mDate] = [
 
 const MeasureSchema = mongoose.Schema(
   {
-    // ID/other measure information
-    id: mString,
-    measure_id: mString,
-    hqmf_id: mString,
-    hqmf_set_id: mString,
-    hqmf_version_number: mNumber,
-    cms_id: mString,
-    title: mString,
-    description: mString,
-    type: mString,
-    category: { type: mString, default: 'Uncategorized' },
+    // A version-specific UUID for the measure
+    hqmf_id: String,
+    // A version-neutral UUID for the measure
+    hqmf_set_id: String,
+    // A Semantic Version-compliant string (e.g. "2.3.4") for the measure
+    hqmf_version_number: String,
+    // A CMS-style string (e.g. "CMS2v4") for the measure
+    cms_id: String,
+    title: String,
+    description: String,
+    // EP/EH, or "unknown"
+    type: String,
+    category: { type: String, default: 'Uncategorized' },
 
     // Measure type variables
-    episode_of_care: mBoolean,
-    continuous_constiable: mBoolean,
-    episode_ids: [],
-
-    // Publishing data (used by Bonnie)
-    published: mBoolean,
-    publish_date: mDate,
-    version: mNumber,
+    episode_of_care: Boolean,
+    continuous_variable: Boolean,
 
     // ELM/CQL Measure-logic related data
     elm_annotations: Mixed,
-    cql: [mString],
+    cql: [String],
```

Comment:
see comment here: https://github.com/projecttacoma/cqm-models/pull/10/files#diff-c25f2a7e44d249869432e81248758e6eR41

Is this going to be updated?